helper-tools/prepare-jamf-policy.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#-
+#- Usage
+#-   $ ./prepare-jamf-policy.sh "/path/to/Install macOS.app"
+#-   Wait seconds, getting checksumm...
+#-
+#-   =====================================================
+#-   Upgrade macOS Parameters:
+#-   Parameter 4: /Applications/Install macOS.app
+#-   Parameter 5: 10.14
+#-   Parameter 6: (Your download trigger policy)
+#-   Parameter 7: 20acadc8d66bb3d882bc0fedf1358c64
+#-   =====================================================
+#-
+#-   Would you need a package archive of Install macOS Mojave.app? [y/n]: y
+#-   Ok, building package archive file of Install macOS Mojave.app. Wait few minutes.
+#-   Done. ~/Desktop/Install macOS Mojave.app.10.14.pkg
+#-   $
+#-
+#-   You can get full-size installer with installinstallmacos.py.
+#-   https://github.com/munki/macadmin-scripts/blob/master/installinstallmacos.py
+#-
+
+if [ $# -eq 0 ]; then
+    # Show help
+    /usr/bin/grep ^#- "$0" | cut -c 4-
+    exit 0
+fi
+
+OSInstaller="$1"
+if [ ! -d "$OSInstaller" ]; then
+    echo "Not found $OSInstaller !"
+    exit 1
+fi
+
+if [ ! -d  "${OSInstaller}/Contents/SharedSupport" ]; then
+    echo "This installer looks like kind of 'stub' installer."
+    /usr/bin/du -sh "${OSInstaller}"
+    echo "Use full size intaller."
+    echo "You can get full-size installer with installinstallmacos.py"
+    echo "https://github.com/munki/macadmin-scripts/blob/master/installinstallmacos.py"
```

Comment:
We can use Mac App Store as well.
We do not have to be limited to the `installinstallmacos.py`.

Reply:
Yes. we can get Installer from App Store. But sometimes it provides 'stub' installer.

If user use an installer which is got from  App Store and  it was full-size one, this block would be skipped.
The other hand, if it is 'stub' one, user should get it from other. So this message is for such kind of case.

Reply:
I understand that it is sometimes stub.
However, the user should also be familiar with retrieving the installer from `installmacos.py`.
Therefore, it is not recommended to specify only that script as main.

Reply:
I see.
I think on it other side. I'll remove such mention from code and then another PR later.

Reply:
I would say it is good to change to that policy overall.